test/unit/mdc-textfield/foundation.test.js

```diff
@@ -173,6 +174,28 @@ test('#init does not add mdc-textfield__label--float-above class if the input do
   td.verify(mockAdapter.addClassToLabel(cssClasses.LABEL_FLOAT_ABOVE), {times: 0});
 });
 
+test('#setHelptextValue has no effect when no help text element is present', () => {
+  const {foundation} = setupTest();
+  foundation.setHelptextValue('foo');
```

Comment:
Hmmmm... I wonder if there's a way to make this a more isolated and specific test. It's not _really_ testing what we're saying it's testing.

Reply:
Good point. I moved that test to the `mdc-text-field.test.js` file.

https://github.com/patrickrodee/material-components-web/blob/37c078df5b21d597076636337e62a1a67494d59c/test/unit/mdc-textfield/mdc-text-field.test.js#L149-L153